src/promptflow-core/promptflow/core/_prompty_utils.py

```diff
@@ -190,6 +190,8 @@ def get_open_ai_client_by_connection(connection, is_async=False):
 def send_request_to_llm(client, api, parameters):
     if api == "completion":
         result = client.completions.create(**parameters)
+    if api == "images":
+        result = client.images.generate(model=parameters['model'], prompt=parameters['messages'][0]['content'], n=1)
```

Comment:
why is it hard coded a n parameter here?  How about other parameters?

Reply:
I haven't added any other parameters because I just wanted something that works for our build demo. These are the 3 main parameters for a quick start. I'll try and in more if I can.